js/logos.json

```diff
@@ -279,8 +275,7 @@
 		"title": "Terra Kulture",
 		"filename": "terra_kulture",
 		"category": [
-			"Ecommerce",
-			"NGO"
+			"Media and Entertainment"
```

Comment:
```suggestion
			"Media",
			"Entertainment"
```

Reply:
This is what I was suggesting.

Reply:
![image](https://user-images.githubusercontent.com/16845280/63218580-c69bf000-c155-11e9-8fac-04b630d591b6.png)
Both are together in the current categorization and separating them may get search broken eventually. However search seems to be fail-safe at the moment since searches are made on strings and not on ID.